core/src/Kokkos_ROCm.hpp

```diff
@@ -47,6 +47,13 @@
 #include <Kokkos_Core_fwd.hpp>
 
 #if defined( KOKKOS_ENABLE_ROCM )
+
```

Comment:
Does this dim3 thing need to be in the public namespace?

Reply:
I think it is public in the cuda compiler.  HCC does not have an equivalent.